packages/cloud_firestore/README.md

```diff
@@ -46,7 +46,7 @@ class BookList extends StatelessWidget {
   Widget build(BuildContext context) {
     return new StreamBuilder(
       stream: Firestore.instance.collection('books').snapshots,
-      builder: (context, snapshot) {
+      builder: (BuildContext context, AsyncSnapshot<QuerySnapshot> snapshot) {
```

Comment:
If you write `StreamBuilder<QuerySnapshot>` as suggested above, then you don't need to put the types there (though it doesn't hurt), since once you have specified the type arguments to the constructor, you've implicitly specified these types as well.